tdrs-backend/tdpservice/stts/test/test_api.py

```diff
@@ -0,0 +1,98 @@
+"""API STT Tests."""
+import pytest
+import logging
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from ...stts.models import STT, Region
+
+User = get_user_model()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler())
+
+
+@pytest.mark.django_db
+def test_stts_is_valid_endpoint(api_client, user):
+    """Test an authorized user can successfully query the STT endpoint."""
+    api_client.login(username=user.username, password="test_password")
+    response = api_client.get(reverse("stts"))
+    assert response.status_code == status.HTTP_200_OK
+
+
+@pytest.mark.django_db
+def test_stts_blocks_unauthorized(api_client, user):
+    """Test an unauthorized user cannot successfully query the STT endpoint."""
+    response = api_client.get(reverse("stts"))
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+def test_stts_alpha_valid_endpoint(api_client, user):
+    """Test an authorized user can successfully query the STT alphabetized endpoint."""
+    api_client.login(username=user.username, password="test_password")
+    response = api_client.get(reverse("stts-alpha"))
+    assert response.status_code == status.HTTP_200_OK
```

Comment:
is there a test that would help validate the difference between the the `stt` and the `stt-alpha` endpoint?